backend/dataall/cdkproxy/assets/lakeformationdefaultsettings/index.py

```diff
@@ -8,6 +8,21 @@ def clean_props(**props):
     return data
 
 
+def validate_principals(principals):
+    iam_client = boto3.client('iam')
+    validated_principals = []
+    for principal in principals:
+        if ":role/" in principal:
+            print(f'Principal {principal} is an IAM role, validating....')
```

Comment:
I have noticed print in multiple places. I think we need to replace it with the logger